traffic-mirroring/README.md

```diff
@@ -0,0 +1,113 @@
+Traffic Mirroring Tool (TMT)
```

Comment:
What does this addition have to do with "Fixing L4 offset for IP packets with Options field"?
Maybe move this to a separate PR?  or else change the PR title?